### PR TITLE
adding root service account paragraph

### PR DESCRIPTION
**Pull request by albertotessarotto**
## Pull Request Type

- [ ] Bugfix
- [ ] Documentation content changes

## PR Checklist

- [ ] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [ ] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [ ] No sensible content has been committed

### CODE REVIEW

File: docs/development_suite/identity-and-access-management/service-account-management.md [hunk 1]
```diff
@@ -148,6 +148,12 @@ A Service Account Role in the Project or any of the Project's Runtime Environmen
   </div>
 </div>
 
+## Managing Service Accounts independent from the Company
+
+It is possible to create Company-independent service accounts in case you need to give them root-level permissions to perform root-level operations.
+
+Only Console Super Users with administrative capabilities can add or delete service accounts independent from the Company, since they are the only ones who can assign root-level permissions to identities. Note that Company-independent service accounts with assigned root-level permissions are only visible and manageable at back-office level, and they will never be visible from a Company Identities portal.
+
```
Comment (marcofilippi) [inline docs/development_suite/identity-and-access-management/service-account-management.md:156]: Maybe it's better to write this down as an info box

File: docs/development_suite/identity-and-access-management/service-account-management.md [hunk 2]
```diff
@@ -148,6 +148,65 @@ A Service Account Role in the Project or any of the Project's Runtime Environmen
   </div>
 </div>
 
+## Managing Service Accounts independent from the Company
+
+It is possible to create Company-independent service accounts in case you need to give them root-level permissions to perform root-level operations.
+
+:::info
+Only Console Super Users with administrative capabilities can add or delete service accounts independent from the Company, since they are the only ones who can assign root-level permissions to identities. Note that Company-independent service accounts with assigned root-level permissions are only visible and manageable at back-office level, and they will never be visible from a Company Identities portal.
+:::
```
Comment (albertotessarotto) [inline docs/development_suite/identity-and-access-management/service-account-management.md:157]:
```suggestion
:::

```

File: docs/development_suite/identity-and-access-management/service-account-management.md [hunk 3]
```diff
@@ -148,6 +148,66 @@ A Service Account Role in the Project or any of the Project's Runtime Environmen
...
+### Creating Root Service Accounts
+
+In order to create a new Root Service Account, you need to contact the `/api/service-accounts` endpoint.
+
+You will need to provide the following parameters in the body:
+
+- `tokenEndpointAuthMethod`: `client_secret_basic` or `private_key_jwt`.
+- `publicKey` (optional): only necessary if you choose `private_key_jwt` as an authentication method. Read [Adding a new Service Account](#adding-a-new-service-account) for further information.
+- `name`: the Root Service Account name.
+- `permissions`: a list of permissions to assign to the Service Account, separed by a comma. See [Console levels and permission management](/docs/development_suite/identity-and-access-management/console-levels-and-permission-management.md) for a reference of the available permissions.
```
Comment (albertotessarotto) [inline docs/development_suite/identity-and-access-management/service-account-management.md:168]:
```suggestion
- `permissions`: a list of permissions to assign to the Service Account, separated by a comma. See [Console levels and permission management](/docs/development_suite/identity-and-access-management/console-levels-and-permission-management.md) for a reference of the available permissions.
```

File: docs/development_suite/identity-and-access-management/service-account-management.md [hunk 4]
```diff
@@ -148,6 +148,66 @@ A Service Account Role in the Project or any of the Project's Runtime Environmen
...
+### Creating Root Service Accounts
+
+In order to create a new Root Service Account, you need to contact the `/api/service-accounts` endpoint.
+
+You will need to provide the following parameters in the body:
+
+- `tokenEndpointAuthMethod`: `client_secret_basic` or `private_key_jwt`.
+- `publicKey` (optional): only necessary if you choose `private_key_jwt` as an authentication method. Read [Adding a new Service Account](#adding-a-new-service-account) for further information.
+- `name`: the Root Service Account name.
+- `permissions`: a list of permissions to assign to the Service Account, separated by a comma. See [Console levels and permission management](/docs/development_suite/identity-and-access-management/console-levels-and-permission-management.md) for a reference of the available permissions.
```
Comment (albertotessarotto) [inline docs/development_suite/identity-and-access-management/service-account-management.md:168]:
```suggestion
- `permissions`: a list of permissions to assign to the Service Account, separated by a comma. See [Console levels and permission management](/development_suite/identity-and-access-management/console-levels-and-permission-management.md) for a reference of the available permissions.
```

File: docs/development_suite/identity-and-access-management/service-account-management.md [hunk 5]
```diff
@@ -148,6 +148,66 @@ A Service Account Role in the Project or any of the Project's Runtime Environmen
   </div>
 </div>
 
+## Managing Service Accounts independent from the Company
+
+It is possible to create Company-independent service accounts in case you need to give them root-level permissions to perform root-level operations.
```
Comment (danielemarostica) [inline docs/development_suite/identity-and-access-management/service-account-management.md:153]:
```suggestion
Root Service Accounts are a special kind of Company-independent Service Accounts with root-level permissions granted and thus can perform root-level operations.
```

File: docs/development_suite/identity-and-access-management/service-account-management.md [hunk 6]
```diff
@@ -148,6 +148,66 @@ A Service Account Role in the Project or any of the Project's Runtime Environmen
...
+{
+  "clientId": "[clientId]",
+  "clientSecret": "[clientSecret]",
+  "clientIdIssuedAt": 1681984988
+}
+```
+
+### Deleting Root Service Accounts
+
+In order to delete a Root Service Account, you need to contact the `/api/service-accounts/{clientId}` endpoint, with the `clientId` obtained from the creation response.
```
Comment (danielemarostica) [inline docs/development_suite/identity-and-access-management/service-account-management.md:198]:
```suggestion
To delete a Root Service Account, you need to contact the `/api/service-accounts/{clientId}` endpoint, providing the `clientId` obtained from the creation response.
```

File: docs/development_suite/identity-and-access-management/service-account-management.md [hunk 7]
```diff
@@ -148,6 +148,66 @@ A Service Account Role in the Project or any of the Project's Runtime Environmen
...
+Example response:
+
+```json
+{
+  "clientId": "[clientId]",
+  "clientSecret": "[clientSecret]",
+  "clientIdIssuedAt": 1681984988
+}
+```
+
```
Comment (danielemarostica) [inline docs/development_suite/identity-and-access-management/service-account-management.md:195]:
```suggestion

:::warning
Remember to store in a secure place both the `clientId` and `clientSecret` as those will be needed to authenticate requests from the Service Account.
:::

```

File: docs/development_suite/identity-and-access-management/console-levels-and-permission-management.md [hunk 8]
```diff
@@ -108,6 +108,7 @@ The following table describes the privileges given to these Console Administrato
 | View all Console resources                          | `console.root.view`                                    |
 | Manage identity Roles, Groups and Bindings          | `console.root.user.bind`                               |
 | Create and delete any user                          | `console.root.user.manage`                             |
+| Create and delete any service account               | `console.root.serviceaccount.manage`                             |
```
Comment (albertotessarotto) [inline docs/development_suite/identity-and-access-management/console-levels-and-permission-management.md:114]:
```suggestion
| Create and delete root service account               | `console.root.serviceaccount.manage`                             |
```

File: docs/development_suite/identity-and-access-management/service-account-management.md [hunk 9]
```diff
@@ -6,7 +6,7 @@ sidebar_label: Manage Service Accounts
 
 Mia-Platform Console allows the creation of Service Accounts, which are typically used for automated processes or to allow a service to access resources on behalf of multiple Users.  
 
-Just like human Users, Service Accounts can be assigned Roles on Company, Project or Runtime Environment level, based on which they will be able to perform different types of action.  
+Just like human Users, Service Accounts can be assigned Roles at Console, Company, Project or Runtime Environment level, based on which they will be able to perform different types of action.
```
Comment (albertotessarotto) [inline docs/development_suite/identity-and-access-management/service-account-management.md:9]:
```suggestion
Just like human users, service accounts can be assigned roles at Console, Company, Project or Runtime Environment levels, according to which they can perform different types of actions.
```

File: docs/development_suite/identity-and-access-management/service-account-management.md [hunk 10]
```diff
@@ -148,6 +148,70 @@ A Service Account Role in the Project or any of the Project's Runtime Environmen
   </div>
 </div>
 
+## Managing Company-independent Service Accounts
+
+Root Service Accounts are a special kind of Company-independent Service Accounts that can be assigned permissions to perform [root-level operations](//development_suite/identity-and-access-management/console-levels-and-permission-management.md#console-root-level-permissions).
```
Comment (albertotessarotto) [inline docs/development_suite/identity-and-access-management/service-account-management.md:153]:
```suggestion
Root service accounts are a special kind of Company-independent service accounts that can be assigned permissions to perform [root-level operations](/development_suite/identity-and-access-management/console-levels-and-permission-management.md#console-root-level-permissions).
```

File: docs/development_suite/identity-and-access-management/service-account-management.md [hunk 11]
```diff
@@ -148,6 +148,70 @@ A Service Account Role in the Project or any of the Project's Runtime Environmen
...
+
+:::info
+Only Console Super Users with administrative capabilities can add or delete service accounts independent from the Company, since they are the only ones who can assign root-level permissions to identities. Note that Company-independent service accounts with assigned root-level permissions are only visible and manageable at back-office level, and they will never be visible from a Company Identities portal.
+:::
+
+### Creating Root Service Accounts
+
+In order to create a new Root Service Account, you need to contact the `/api/service-accounts` endpoint.
```
Comment (albertotessarotto) [inline docs/development_suite/identity-and-access-management/service-account-management.md:161]:
```suggestion
In order to create a new Root service account, you need to contact the `/api/service-accounts` endpoint.
```

File: docs/development_suite/identity-and-access-management/service-account-management.md [hunk 12]
```diff
@@ -148,6 +148,70 @@ A Service Account Role in the Project or any of the Project's Runtime Environmen
...
+
+You will need to provide the following parameters in the body:
+
+- `tokenEndpointAuthMethod`: `client_secret_basic` or `private_key_jwt`.
+- `publicKey` (optional): only necessary if you choose `private_key_jwt` as an authentication method. Read [Adding a new Service Account](#adding-a-new-service-account) for further information.
+- `name`: the Root Service Account name.
+- `permissions`: a list of permissions to assign to the Service Account, separated by a comma. See [Console levels and permission management](/development_suite/identity-and-access-management/console-levels-and-permission-management.md) for a reference of the available permissions.
```
Comment (albertotessarotto) [inline docs/development_suite/identity-and-access-management/service-account-management.md:168]:
```suggestion
- `permissions`: a list of permissions to assign to the service account, separated by a comma. See [Console levels and permission management](/development_suite/identity-and-access-management/console-levels-and-permission-management.md) for a reference of the available permissions.
```

File: docs/development_suite/identity-and-access-management/service-account-management.md [hunk 13]
```diff
@@ -148,6 +148,70 @@ A Service Account Role in the Project or any of the Project's Runtime Environmen
...
+
+To authenticate your request, you will need to provide your access token and the Console client key.
+
+Here is an example of cURL request for creating a Root Service Account with the `client_secret_basic` authentication method:
```
Comment (albertotessarotto) [inline docs/development_suite/identity-and-access-management/service-account-management.md:172]:
```suggestion
Here is an example of cURL request for creating a Root service account with the `client_secret_basic` authentication method:
```

**Pull request merged**